File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/relationshipsvb/vb/workwithrelationships5.vb
' <snippetworkwithrelationships5>


 Dim deleteOneToManyRelationshipRequest As DeleteRelationshipRequest =
  New DeleteRelationshipRequest With {
   .Name = "new_account_campaign"
  }

 _serviceProxy.Execute(deleteOneToManyRelationshipRequest)

 Dim deleteManyToManyRelationshipRequest As DeleteRelationshipRequest =
  New DeleteRelationshipRequest With {
   .Name = "new_accounts_campaigns"
  }

 _serviceProxy.Execute(deleteManyToManyRelationshipRequest)

 Console.WriteLine("Entity Relationships have been deleted.")

' </snippetworkwithrelationships5>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/relationships/cs/workwithrelationships5.cs
// <snippetworkwithrelationships5>


 DeleteRelationshipRequest deleteOneToManyRelationshipRequest = 
     new DeleteRelationshipRequest
 {                    
     Name = "new_account_campaign"
 };

 _serviceProxy.Execute(deleteOneToManyRelationshipRequest);

 DeleteRelationshipRequest deleteManyToManyRelationshipRequest = 
     new DeleteRelationshipRequest
 {
     Name = "new_accounts_campaigns"
 };

 _serviceProxy.Execute(deleteManyToManyRelationshipRequest);

 Console.WriteLine("Entity Relationships have been deleted.");

// </snippetworkwithrelationships5>